problem1.ts

```diff
@@ -0,0 +1,26 @@
+// type partFuncOutput = (array: Array<number>) => number | number;
+
+export function sum(args: Array<number>): number {
+  // console.log(args)
+  return args.reduce((akum, next) => akum + next, 0);
+}
+
+export function partFunc(
+  fn: (array: Array<number>) => number,
+  ...argsInput: number[]
```

Comment:
Please choose and stick to an array type definition syntax to keep the code consistent